objax/nn/layers.py

```diff
@@ -187,6 +203,13 @@ def __call__(self, x: JaxArray) -> JaxArray:
             y += self.b.value
         return y
 
+    def __repr__(self):
+        args = dict(nin=self.w.value.shape[2] * self.groups, nout=self.w.value.shape[3], k=self.w.value.shape[:2],
```

Comment:
use `self.nin` instead of `self.w.value.shape[2] * self.groups`?
Or maybe remove `self.nin` completely if it's actually not used anywhere

Reply:
Removed `self.nin` to avoid incoherencies.